CHANGES.txt

```diff
@@ -4,8 +4,11 @@ Changes
 2.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- compatibility for python 3
+- update code to work with python2 and python3
+- adjust egg to work with newer version of setuptools
```

Comment:
I'm looking through my last commits and can't remember me why I wrote this line. I think it's wrong and will be better to remove it. The only changes in the setup.py are the classifier and a additional attribute test_suite. 

By the way, I just saw the README of this package has the prefix ".txt". In github it would looking better with the prefix ".rst". If you want, I will rename this file.